docs/source/glossary.md

```diff
@@ -17,4 +17,20 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
-:::::
+
+[Equidispersion](http://www.ce.memphis.edu/7012/L20_CountDataModels_v2.pdf)
+  If in a Poisson distribution if the variance equals the mean of the distribution, it is reffered to as equidispersion.
+
+[Generalized Poisson PMF](https://www.sciencedirect.com/science/article/pii/S0047259X14000256)
+  A generalization of the {term}`Poisson distribution`, with two parameters X1, and X2, is obtained as a limiting form of the {term}`generalized negative binomial distribution`. The variance of the distribution is greater than, equal to or smaller than the mean according. as X2 is positive, zero or negative. For formula and more detail visit the link 
+
+[Bayes' theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem)
+  Describes the probability of an event, based on prior knowledge of conditions that might be related to the event. For example, if the risk of developing health problems is known to increase with age, Bayes' theorem allows the risk to an individual of a known age to be assessed more accurately (by conditioning it on their age) than simply assuming that the individual is typical of the population as a whole.
```

Comment:
I would add the formula and some lines explaining what the formula means.

Reply:
I have written the formula for Bayes' theorem but I have 2 problems:
1) I don't exactly know any way to write the formula, the way it is written by hand
2) I think the formula for PMF would be a little complex to put in text thus I feel that we should just allow the reader to visit the link to read the formula

Reply:
See if the way that I've written formula for Bayes' works or not and the please let me know